ui/styl/modules/palette.styl

```diff
@@ -56,3 +56,9 @@ $secondary-purple = #D77FBF // Chart Color
 $secondary-slate-gray = #7C8495 // Secondary Navigation
 $secondary-light-gray = #A6A6B2 // Secondary Headers
 
+// not part of official palette
```

Comment:
nit: not sure what comment style we're following, but right now this file is a mixed bag.